packages/tradershub/src/components/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -65,13 +64,17 @@ const CurrencySwitcher = () => {
             <div className='flex-none '>{IconToCurrencyMapper[iconCurrency].icon}</div>
             <div className='grow'>
                 <Text
-                    bold={isDemo}
                     className={isDemo ? 'text-status-light-information' : 'text-system-light-less-prominent-text'}
                     size='sm'
+                    weight={isDemo ? 'bold' : 'normal'}
                 >
                     {isDemo ? activeAccount.display_balance : IconToCurrencyMapper[iconCurrency].text}
                 </Text>
-                <Text bold={!isDemo} className={!isDemo ? 'text-status-light-success' : undefined} size='sm'>
+                <Text
+                    className={!isDemo ? 'text-status-light-success' : undefined}
```

Comment:
This is correct however I feel a better way of writing this is:
```suggestion
                    className={qtMerge(!isDemo ? 'text-status-light-success' : '')}
```
what do you think?

Reply:
@aizad-deriv Honestly, I'm not sure we need `qtMerge` here, because we add just one class. WE have to use qtMerge when we want to combine some tailwind styles.